dex-it/src/test/scala/com/wavesplatform/it/matcher/api/http/rates/GetRatesSpec.scala

```diff
@@ -0,0 +1,37 @@
+package com.wavesplatform.it.matcher.api.http.rates
+
+import com.typesafe.config.{Config, ConfigFactory}
+import com.wavesplatform.dex.api.http.entities.HttpRates
+import com.wavesplatform.dex.domain.asset.Asset.Waves
+import com.wavesplatform.dex.it.api.RawHttpChecks
+import com.wavesplatform.it.MatcherSuiteBase
+
+class GetRatesSpec extends MatcherSuiteBase with RawHttpChecks {
+
+  val defaultRates: HttpRates = Map(Waves -> 1d)
+
+  override protected def dexInitialSuiteConfig: Config = ConfigFactory.parseString(
+    s"""waves.dex {
+       |  price-assets = [ "$BtcId", "$UsdId", "WAVES" ]
+       |}""".stripMargin
+  )
+
+  override protected def beforeAll(): Unit = {
+    wavesNode1.start()
+    broadcastAndAwait(IssueBtcTx, IssueUsdTx)
+    dex1.start()
+  }
+
+  "GET /matcher/settings/rates " - {
```

Comment:
```
rates " - {
```
please remove the space after "rates": `rates" - {`. It could lead to strange errors if we try to run this test suite locally and specify the test's name like: `dex-it/testOnly *GetRatesSpec  -- -z "should return default"`

Reply:
ok